cmake/std/atdm/mutrino/environment.sh

```diff
@@ -0,0 +1,57 @@
+################################################################################
+#
+# Set up env on mutrino for ATMD builds of Trilinos
+#
+# This source script gets the settings from the JOB_NAME var.
+#
+################################################################################
+
+echo "Using mutrino compiler stack $ATDM_CONFIG_COMPILER to build $ATDM_CONFIG_BUILD_TYPE code with Kokkos node type $ATDM_CONFIG_NODE_TYPE"
+export ATDM_CONFIG_BUILD_COUNT=8
+export OMPI_CXX=
+export ATDM_CONFIG_USE_NINJA=OFF
+export OMP_NUM_THREADS=2
+
+# Don't purge as this removes the Cray default environment
+#module purge
+module use /projects/EMPIRE/mutrino/tpls/hsw/modulefiles
+
+# Use srun to run mpi jobs
+export ATDM_CONFIG_MPI_EXEC="/opt/slurm/bin/srun"
+
+# srun does not accept "-np" for # of processors
+export ATDM_CONFIG_MPI_EXEC_NUMPROCS_FLAG="-n"
+export ATDM_CONFIG_KOKKOS_ARCH=HSW
+export ATDM_CONFIG_CTEST_PARALLEL_LEVEL=16
+
+if [ "$ATDM_CONFIG_COMPILER" == "INTEL" ]; then
+    module load devpack/20180124/cray/7.6.2/intel/17.0.4
+    module load gcc/4.9.3
+    module load cmake/3.9.0
```

Comment:
We should try to install cmake 3.10+ on this system.  That will be needed to allow us to switch over to the all-at-once mode once we make the switch to the new testing-vm.sandia.gov CDash site.

But we can do that later.  We don't need to block this PR for that.